packages/aws-cdk-lib/aws-cloudfront-origins/lib/function-url-origin.ts

```diff
@@ -0,0 +1,49 @@
+import { validateSecondsInRangeOrUndefined } from './private/utils';
+import * as cloudfront from '../../aws-cloudfront';
+import * as lambda from '../../aws-lambda';
+import * as cdk from '../../core';
+
+/**
+ * Properties for a Lambda Function URL Origin.
+ */
+export interface FunctionUrlOriginProps extends cloudfront.OriginProps {
+  /**
+   * Specifies how long, in seconds, CloudFront waits for a response from the origin.
+   * The valid range is from 1 to 180 seconds, inclusive.
```

Comment:
I'm seeing the max read timeout as 60 seconds https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-cloudfront-distribution-customoriginconfig.html#cfn-cloudfront-distribution-customoriginconfig-originreadtimeout. Where does the 180 seconds come from?

Reply:
This is in reference to this code; I added this because Note was missing.

https://github.com/aws/aws-cdk/blob/b4daf84cf8a90ef38e3a2328df6271bdaf19de01/packages/aws-cdk-lib/aws-cloudfront-origins/lib/http-origin.ts#L37-L58

Reply:
Thanks for the clarification!